docs/source/ReleaseNotes.md

```diff
@@ -16,9 +18,29 @@
 * [July 2020](#july-2020)
 * [May 2020](#may-2020)
 
+## December 2021
+
+* Internal improvements to our Config API. Added functionality to integrate with JW Player systems.
+
+* Internal improvements to our CPIX API.
+
+## November 2021
+
+* Improvements to VUDRM Admin. Video.js and Dash.js added as an option for the public player.
```

Comment:
Should this be Studio DRM admin now?

Reply:
Probably not - the site, URLs and such haven't been rebranded, so could add confusion?

Happy to do it if you like though.

Reply:
@askolodny Any opinions on this? Do we want to leave it as "VUDRM Admin" for now and then when all of these docs are actually moved over change it to "Studio DRM" then?